prime-router/build.gradle.kts

```diff
@@ -107,6 +108,13 @@ compileTestKotlin.kotlinOptions.jvmTarget = "11"
 tasks.clean {
     // Delete the old Maven build folder
     delete("target")
+    // clean up all the old event files in the SOAP set up
+    doLast {
+        val eventsDir = File("../.environment/soap_service/soap/event/v1/")
+        FileUtils.listFiles(eventsDir, arrayOf("event"), true).forEach {
+            it.delete()
+        }
+    }
```

Comment:
As part of the clean task, I'm asking Gradle to delete all the *.event files in the `event/v1` folder created by CastleMock. They're just an audit log and we don't need to keep them around permanently.